packages/insured-bridge-relayer/CHANGELOG.md

```diff
@@ -1,307 +0,0 @@
-# Change Log
```

Comment:
I think we still run the dispute bot in prod since there are funds remaining in the protocol. I believe merging this as is will cause those bots to break.

I think the simplest strategy is to propose a liveness extension in Across v1 to 5200 weeks (100 years), which would effectively allow us to turn off the dispute bots.

Thoughts? cc @nicholaspai

Reply:
yup! that's an option. other option would be to just run the hup on a pinned version of this code.